src/routes/_components/dialog/components/MediaDialog.html

```diff
@@ -77,16 +93,25 @@
             </li>
         </ul>
       {/if}
-      {#if canPinchZoom}
+      {#if canPinchZoom }
         <IconButton
           className="media-control-button"
           svgClassName="media-control-button-svg"
-          pressable={true}
+          pressable={canPinchZoom}
```

Comment:
Same here, this can just be `pressable={true}`.

```suggestion
          pressable={true}
```